apps/frontend/src/assets/css/form.css

```diff
@@ -0,0 +1,47 @@
+.form-field {
```

Comment:
Component specific styles will be added separately for components, not as global styles